Web_Development/Javascript/Topics/1.Hello_World/README.md

```diff
@@ -1 +1,174 @@
-# To Do
\ No newline at end of file
+# Say "Hello! World" in JavaScript - For Beginners
+
+<br>
+<br>
+
+Hello! every one . This tutorial is about writing a simple "Hello! World" program, which is normally the beginning of coding in any new language. 
+
+There are many possible ways to do so in Javascript. Here is a list of beginner-friendly steps you need follow in order to print "Hello! World" in Javascript. So, let's begin with 5 easy steps.
+
+<br>
+
+
+## STEP 1: Download an editor to write code:
+
+You can also use Notepad for this purpose. It becomes quite easy to write our code in a smart editor like Visual Studio Code etc. So, it is a recommended to download and install a good editor before you start to write the code. 
+
+<br>
+
+## STEP 2: Create a new file with .html extention:
+
+After installing the editor we simply need to open the editor and create a new file and save it with .html extention. For example, your file name may look like 'Hello_World.html' .
+This will enable us to run our code on browser.
+
+<br>
+
+## STEP 3: Open the file created in STEP 2 to write some code:
+
+Open the file that we created in step 2 above to write the following code in it:
+
+<br>
+
+    <html>
+    <head>
+       <title>Hello World In JavaScript</title>
+    </head>
+    <body>
+        <script>
+            console.log('Hello!World');
+        </script> 
+        
+        <!-- the script enclosed in <script></script>is the actual Javascript code we need to execute in order to print Hello! World. -->
+        
+    </body>
+    </html>
+
+<br>
```

Comment:
If possible, enclose code-blocks in the Markdown syntax meant to enclose code. That will make it much easier for the reader to see syntax-highlighted code.